api/MonitorizareVot.Ong/src/MonitorizareVot.Api/Queries/PollingStationCreateHandler.cs

```diff
@@ -0,0 +1,50 @@
+using System.Threading;
+using System.Threading.Tasks;
+using AutoMapper;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using MonitorizareVot.Domain.Ong.Models;
+using MonitorizareVot.Ong.Api.ViewModels;
+using System.Linq;
+using MonitorizareVot.Api.Exceptions;
+using System;
+
+namespace MonitorizareVot.Api.Queries
+{
+    public class PollingStationCreateHandler : IRequestHandler<PollingStationViewCommand, int>
+    {
+        private readonly VoteMonitorContext _context;
+        private readonly ILogger _logger;
+        private readonly IMapper _mapper;
+
+        public PollingStationCreateHandler(VoteMonitorContext context, ILogger logger, IMapper mapper)
+        {
+            this._context = context;
+            this._logger = logger;
+            this._mapper = mapper;
+        }
+        public Task<int> Handle(PollingStationViewCommand request, CancellationToken cancellationToken)
+        {
+            Random random = new Random();
+
+            var pollingStation = _mapper.Map<PollingStation>(request.PollingStationView);
+            var maxId = _context.PollingStations.Max(p => p.Id);
+            County pollingStationCounty = _context.Counties
+                            .Where(c => c.Code == request.PollingStationView.CountyCode)
+                            .FirstOrDefault();
+
+            if(pollingStationCounty == null)
+                throw new ResourceNotFoundException();
+
+            pollingStation.Id = ++maxId;
+            pollingStation.County = pollingStationCounty;
+            pollingStation.IdCounty = pollingStationCounty.Id;
+            pollingStation.TerritoryCode = random.Next(10000).ToString();
```

Comment:
Any reason why a generated GUID would not work here? If the function is called often and you want the TerritoryCode to be unique, then Random.Next() is not the best option so this might raise issues in the future. 
(I don't know all the details so I'm just leaving this as a suggestion, it will not block the PR)

Reply:
The TeritoryCode will probably be removed from the entity domain model because right now it does not make sense to keep it, but this will be done as part of a complete rewrite of the datamodel of the solution.